nuget/Dockerfile

```diff
@@ -1,7 +1,44 @@
 FROM ghcr.io/dependabot/dependabot-updater-core
 
-USER dependabot
+USER root
+
+ENV DEPENDABOT_NATIVE_HELPERS_PATH="/opt"
+
+# Install .NET SDK dependencies
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  libicu-dev \
```

Comment:
We should considering pinning this version to make the image more reproducible

Reply:
done, will push changes shortly